### DISCUSSION TITLE
Documentation search could be better

### DISCUSSION THREAD

**Discussion by rtconner** [tailwindlabs/tailwindcss#14295] [General]
The search on tailwindcss.com always makes me sad.

Just one example:
Searching for "colors" does not bring up the "Colors" list as an item (https://tailwindcss.com/docs/customizing-colors)
Instead I get a lot of pages that are not the list of colors:

<img width="779" alt="Screenshot 2024-08-29 at 3 56 04 PM" src="https://github.com/user-attachments/assets/8d41d7b3-3ee9-47c4-90d4-f090cb08e427">

**Comment by kaizenics** [comment]
too lazy to navigate in the sidebar?
